Packs/AzureWAF/Integrations/AzureWAF/AzureWAF.py

```diff
@@ -27,9 +27,18 @@
 
 class AzureWAFClient:
     @logger
-    def __init__(self, app_id, subscription_id, resource_group_name, verify, proxy,
+    def __init__(self, app_id, subscription_id, resource_group_name, verify, proxy, auth_type, tenant_id=None,
+                 enc_key=None, auth_code=None, redirect_uri=None,
                  azure_ad_endpoint: str = 'https://login.microsoftonline.com'):
-
+        AUTH_TYPES_DICT: dict = {'User Auth': {
```

Comment:
can we format a bit differently so its clearer what the keys are? like
```
{
    key1 : { ...
           ...
           ...},
    key2: { ...
           ...
           ...}
 }
```
 if the linters let